src/components/NewCreditCardForm.js

```diff
@@ -78,9 +78,13 @@ class NewCreditCardFormWithoutStripe extends React.Component {
 const NewCreditCardFormWithStripe = injectStripe(NewCreditCardFormWithoutStripe);
 
 const NewCreditCardForm = props => (
-  <Elements>
+  <Elements locale={props.locale}>
     <NewCreditCardFormWithStripe {...props} />
```

Comment:
If we're intentionally passing `locale` to `NewCreditCardFormWithStripe` then we should add it to its prop types. Otherwise maybe we should omit the value.